lib/Capabilities/CapabilitiesDataType.js

```diff
@@ -21,6 +22,9 @@ const CapabilitiesDataType = ({ content, readOnly, onChangeCapabilityCheckbox, i
    * show only checkbox for that particular action.
    * If readOnly mode we should show the checkbox for all actions.
    */
+
+
+
```

Comment:
it is better if you remove the extra empty lines

Reply:
Removed! Thanks a lot